src/provisioner/log.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/env python3
+
+# CORTX Python common library.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+from cortx.utils.log import Log
+
+
+class CortxProvisionerLog(Log):
```

Comment:
Can you add elaborate comments here, about initliazing Cortx utils logger only, though based on config, to log in log file & console, etc.

Reply:
Elaborated in class docstring and init() method.

Reply:
Done.